_api-reference/index-apis/resolve.md

```diff
@@ -0,0 +1,94 @@
+---
+layout: default
+title: Resolve index
+parent: Index APIs
+nav_order: 63
+---
+
+# Resolve index
+
+Introduced 1.0
+{: .label .label-purple }
+
+The Resolve Index API lists the indexes, aliases, and data streams based on the specified name. Multiple patterns and remote clusters are supported.
```

Comment:
I'm not understanding what this means... Is it something like this, "You use the Resolve Index API to return a list of indexes, aliases, and data streams that match a specified pattern." ?